src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/VariableDeclarationUsageDistanceCheck.java

```diff
@@ -412,37 +412,47 @@ public void visitToken(DetailAST ast) {
                 else {
                     entry = calculateDistanceInSingleScope(semicolonAst, variable);
                 }
-                final DetailAST variableUsageAst = entry.getKey();
-                final int dist = entry.getValue();
-                if (dist > allowedDistance
-                        && !isInitializationSequence(variableUsageAst, variable.getText())) {
-                    if (ignoreFinal) {
-                        log(ast, MSG_KEY_EXT, variable.getText(), dist, allowedDistance);
-                    }
-                    else {
-                        log(ast, MSG_KEY, variable.getText(), dist, allowedDistance);
-                    }
-                }
+                logViolation(ast, variable, entry);
```

Comment:
this method does not actually do only log violation.
lets make code `if (isViolation(....)) { log(....);}`

Reply:
@Vyom-Yadav , please do

Reply:
Done.